src/telegraf.ts

```diff
@@ -69,30 +59,21 @@ namespace Telegraf {
   }
 }
 
-const allowedUpdates: tt.UpdateType[] | undefined = undefined
-
 export class Telegraf<
   TContext extends Context = Context
 > extends Composer<TContext> {
   private readonly options: Telegraf.Options<TContext>
   private webhookServer?: http.Server | https.Server
+  private polling?: Polling
   /** Set manually to avoid implicit `getMe` call in `launch` or `webhookCallback` */
   public botInfo?: tt.UserFromGetMe
   public telegram: Telegram
   readonly context: Partial<TContext> = {}
-  private readonly polling = {
-    allowedUpdates,
-    limit: 100,
-    offset: 0,
-    started: false,
-    stopCallback: noop,
-    timeout: 30,
-  }
 
-  private handleError: (err: unknown, ctx: TContext) => void = (err: any) => {
-    console.error()
-    console.error((err.stack ?? err.toString()).replace(/^/gm, '  '))
-    console.error()
+  private handleError = async (err: unknown, ctx: TContext): Promise<void> => {
+    process.exitCode = 1
```

Comment:
This does not belong into `telegraf` itself, it should be the bot that determines its exit codes.

However, it certainly does belong into `bin/telegraf`, so maybe we should move this line there.

Reply:
This tries to emulate `warn-with-error-code` [unhandled rejections mode](https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode) when `warn` mode is used (default before Node 15). It effectively changes nothing in `throw`, `strict`, and `warn-with-error-code` modes. Without this, the process will cleanly exit with status code 0, perhaps fooling process manager into not restarting the process. Any better ideas?

Reply:
```suggestion
    // set exit code to emulate `warn-with-error-code` behavior of
    // https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode
    // to prevent a clean exit despite an error being thrown
    process.exitCode = 1
```